_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/consume-topic-key-value.adoc

```diff
@@ -4,13 +4,13 @@ View the messages being produced to the Kafka topic in Confluent Cloud.
 There are many ways to do this, including the Confluent Cloud UI, but for this tutorial we will show you how to it with the Confluent Cloud CLI.
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/harness-console-consumer-keys.sh %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/harness-console-consumer-keys.sh %}</code></pre>
 +++++
 
 After the consumer starts, you should see the following output in a few seconds:
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/consume-topic_expected.log %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/consume-topic_expected.log %}</code></pre>
 +++++
 
 When you are done, type `Ctrl+C`.
```

Comment:
Should we show the user how to destroy the connector? It's an hourly billable item so I think we should cover that base.

Reply:
Yes, good idea.  I'll add that step.

Reply:
@rspurgeon Addressed in latest commit